packages/frontend-2/components/automate/function/page/ParametersDialog.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <LayoutDialog v-model:open="open" max-width="md" title="Function Parameters">
+  <LayoutDialog
+    v-model:open="open"
+    :buttons="dialogButtons"
+    max-width="md"
+    title="Function parameters"
+  >
     <div class="flex flex-col space-y-4">
       <template v-if="finalParams">
         <FormJsonForm :schema="finalParams" />
-        <LayoutDialogSection
-          title="Parameter schema"
-          :icon="BeakerIcon"
-          border-t
-          border-b
-        >
+        <LayoutDialogSection title="Parameter schema" border-t border-b>
           <FormTextArea
             name="actionYaml"
             readonly
             :model-value="JSON.stringify(finalParams, null, 2)"
-            class="text-sm text-primary"
+            textarea-classes="!bg-foundation !border border-outline-2 p-2 rounded-lg font-mono !font-body-2xs"
```

Comment:
I haven't seen `textarea-classes` before 🤔. Is this a real prop?

Reply:
Yup it is, it's used only once or twice I think, but inside `FormTextArea` is a nested `textarea` element, this prop allows you to set the classes on that element :)

Reply:
Ah!